### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by rick-kenna**
Code to automatically launch a connector.

### CODE REVIEW

File: python/connectors/connectors_auto_start.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+import sys
+import json
+import requests
+from datetime import datetime, timedelta
+from prettytable import PrettyTable
+
+# Returns connector information from the List Connectors API.
+def get_connectors(base_url, headers):
+    connectors = []
+    list_connectors_url = base_url + "connectors"
```
Comment (nhenry012) [inline python/connectors/connectors_auto_start.py:13]: This seems to mix and match string concatenation and interpolation. I would say we should switch this line to use interpolation the same way it is used on line 17. I.E. f"{base_url}connectors".

File: python/connectors/connectors_auto_start.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        sys.exit(1)
+    
+    resp_json = response.json()
+    connectors = resp_json['connectors']
+
+    return connectors
+
+# Gets the connector runs for a the specified connector ID.
+def get_connector_runs(base_url, headers, connector_id):
+    connector_runs = []
```
Comment (nhenry012) [inline python/connectors/connectors_auto_start.py:27]: I think this is a copy paste thing. This variable does not appear to be used.

File: python/connectors/show_connector_status.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
+# Checks if connector has run in the last 24 hours.  If not starts it up.
+
+import os
+import sys
+import json
+import requests
+from datetime import datetime, timedelta
+
+# Returns connector information from the List Connectors API.
+def get_connectors(base_url, headers):
```
Comment (nhenry012) [inline python/connectors/show_connector_status.py:10]: Duplicated function? Could we make "helper functions" file to contain this?

**Review by nhenry012 — CHANGES_REQUESTED** [review]
Looks great! Good work. I only have a handful of notes.

File: python/connectors/show_connector_status.py (hunk 3 above)
Comment (rick-kenna) [reply python/connectors/show_connector_status.py:10]: Yeah, good idea.

File: python/connectors/connectors_auto_start.py (hunk 2 above)
Comment (rick-kenna) [reply python/connectors/connectors_auto_start.py:27]: Removed.  The function started out to do something a little different and this variable was left behind.

File: python/connectors/connectors_auto_start.py (hunk 1 above)
Comment (rick-kenna) [reply python/connectors/connectors_auto_start.py:13]: I fixed all the string concatenations I could find.

**Pull request merged**